tests/test_input_damages.py

```diff
@@ -0,0 +1,1287 @@
+    calculate_labor_damages,
+    compute_ag_damages,
+    read_energy_files,
+    read_energy_files_parallel,
+    calculate_energy_impacts,
+    concatenate_energy_damages,
+    calculate_energy_batch_damages,
+    calculate_energy_damages,
+    prep_mortality_damages,
+    coastal_inputs,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def test_parse_projection_filesys(tmp_path):
+    """
+    Test that parse_projection_filesys correctly retrieves projection system output structure
+    """
+    rcp = ["rcp85", "rcp45"]
+    gcm = ["ACCESS1-0", "GFDL-CM3"]
+    model = ["high", "low"]
+    ssp = [f"SSP{n}" for n in range(2, 4)]
+    batch = ["batch6", "batch9"]
+
+    for b in batch:
+        for r in rcp:
+            for g in gcm:
+                for m in model:
+                    for s in ssp:
+                        os.makedirs(os.path.join(tmp_path, b, r, g, m, s))
+
+    out_expected = {
+        "batch": list(chain(repeat("batch9", 16), repeat("batch6", 16))),
+        "rcp": list(chain(repeat("rcp85", 8), repeat("rcp45", 8))) * 2,
+        "gcm": list(chain(repeat("ACCESS1-0", 4), repeat("GFDL-CM3", 4))) * 4,
+        "model": list(chain(repeat("high", 2), repeat("low", 2))) * 8,
+        "ssp": ["SSP2", "SSP3"] * 16,
+        "path": [
+            os.path.join(tmp_path, b, r, g, m, s)
+            for b in ["batch9", "batch6"]
+            for r in rcp
+            for g in gcm
+            for m in model
+            for s in ssp
+        ],
+        "exists": [True] * 32,
+        "iam": list(chain(repeat("OECD Env-Growth", 2), repeat("IIASA GDP", 2))) * 8,
+    }
+    df_out_expected = pd.DataFrame(out_expected)
+
+    df_out_actual = _parse_projection_filesys(input_path=tmp_path)
+    df_out_actual.reset_index(drop=True, inplace=True)
+
+    pd.testing.assert_frame_equal(df_out_expected, df_out_actual)
+
+
+@pytest.fixture
+def labor_in_val_fixture(tmp_path):
+    """
+    Create fake labor input data for tests and save to netcdf
+    """
+    ds_in_val = xr.Dataset(
+        {
+            "regions": (["region"], np.array(["ZWE.test_region", "USA.test_region"])),
+            "rebased": (["year", "region"], np.full((4, 2), 2)),
+        },
+        coords={
+            "year": (["year"], [2009, 2010, 2099, 2100]),
+        },
+    )
+
+    rcp = ["rcp45", "rcp85"]
+    gcm = ["ACCESS1-0", "GFDL-CM3"]
+    model = ["low", "high"]
+    ssp = ["SSP3"]
+    batch = ["batch" + str(i) for i in range(0, 15)]
+
+    for b in batch:
+        for r in rcp:
+            for g in gcm:
+                for m in model:
+                    for s in ssp:
+                        d = os.path.join(tmp_path, "labor_in", b, r, g, m, s)
+                        if not os.path.exists(d):
+                            os.makedirs(d)
+                        infile_val = os.path.join(
+                            d, "uninteracted_main_model-wage-levels.nc4"
+                        )
+                        ds_in_val.to_netcdf(infile_val)
```

Comment:
I'm wondering if there should be any variation to what we save out here, for each batch, rcp, gcm, ... Will we miss anything in the tests if the data are all the same? Similar question for histclim below. And it looks like it's the same as here too. I see it makes testing `test_calculate_labor_impacts` easier tho

Reply:
The main concern about adding variations to input damages is that it could make outputs hard to predict since some functions have subtraction and conversion steps. Given that most functions get tested here are straightforward and it doesn't seem like we have missed anything, I think it might be cost-effective to keep all data the same.